app/views/nodes/index.html.slim

```diff
@@ -48,15 +47,13 @@ h1 Nodes
           td.text-center
             - if m.role == "master"
               = radio_button_tag "roles[master]", m.id, true, disabled: true
-            - elsif Minion.all.collect(&:role).include?("master")
+            - elsif Minion.exists?(role: "master")
               = radio_button_tag "roles[master]", m.id, false, disabled: true
             - else
               = radio_button_tag "roles[master]", m.id
   .clearfix.text-right.steps-container
     a.btn.btn-danger[href="/"] Back
-    - if Minion.all.collect(&:role).include?("master")
-      = submit_tag "Apply", class: "btn btn-primary", disabled: true
+    - if Minion.exists?(role: "master")
+      = submit_tag "Bootstrap Cluster", class: "btn btn-primary", disabled: true
```

Comment:
I think, UX wise, the correct thing to do would be to not allow the user to click on a button that will result in an error. Or maybe, show a client side message that will explain why this action is not permitted. On the other hand, since we are not the UX experts here, I would return a friendlier message to the user, not the exception message. E.g. "Please spawn some minions before you try to bootstrap a cluster" or something like that.